src/features/classification/_mixins.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../styles/variables';
+
+@mixin bdl-ClassificationBadge {
+    align-items: center;
+    border: 1px solid;
+    border-radius: $bdl-border-radius-size;
+    display: inline-flex;
+    margin: 0;
+    padding: 2px 5px;
+}
+
+@mixin bdl-ClassificationBadge-name {
+    font-size: 10px !important; // To inhibit overriding
```

Comment:
Can we avoid this? Overriding by whom? The parent application?

Reply:
yeah, see other comment